lib/tests/CMakeLists.txt

```diff
@@ -29,14 +30,26 @@ endif()
 # described in https://gcc.gnu.org/ml/gcc-help/2015-08/msg00035.html, we get
 # the error undefined reference to symbol '__tls_get_addr@@GLIBC_2.3'.
 # Build mock_server as a separate shared library to avoid this error.
+add_library(pcp-chunks  # pcp-chunk is needed by MockServer (no threads)
+    ${SOURCE_DIR}/protocol/chunks.cc
+    ${SOURCE_DIR}/protocol/message.cc
+    ${SOURCE_DIR}/protocol/schemas.cc
+    ${SOURCE_DIR}/protocol/serialization.cc
+    ${SOURCE_DIR}/validator/schema.cc
+    ${SOURCE_DIR}/validator/validator.cc)
+target_link_libraries(pcp-chunks PRIVATE
```

Comment:
This will need to declare export symbols for those files, or explicitly build pcp-chunks as a static library. The Leatherman symbol_exports helper should work for the 1st option (and can also now be used for libcpp-pcp-client).